internal/registry/revisions.go

```diff
@@ -19,9 +19,12 @@ import (
 
 // SearchOptions specify the query parameters without the package name for the search API
 type SearchOptions struct {
-	Experimental  bool   `url:"experimental"`
+	Prerelease    bool   `url:"prerelease"`
```

Comment:
... oh, and on the EPR release.

Reply:
Yes, I think this will be the last piece to merge.